core/src/main/java/bisq/core/api/CoreOffersService.java

```diff
@@ -67,6 +68,16 @@ public CoreOffersService(CreateOfferService createOfferService,
         this.user = user;
     }
 
+    Offer getOffer(String id) {
+        List<Offer> offers = offerBookService.getOffers().stream()
+                .filter(o -> o.getId().equals(id))
+                .collect(Collectors.toList());
```

Comment:
As we expect only one I would use a findAny and create an optional instead of a list.

Reply:
Ah see you changed that in 8394bd1f6401a0b6f6843bd3b31e3c3b391d7f3c ;-)